tests/core/recipes/onetep_recipes/test_onetep.py

```diff
@@ -0,0 +1,39 @@
+from ase.build import molecule
+from quacc import SETTINGS
+from pathlib import Path
+from quacc.recipes.onetep.core import static_job
+from quacc.utils.files import copy_decompress_files
+
+DEFAULT_SETTINGS = SETTINGS.model_copy()
+DATA_DIR = Path(__file__).parent / "data"
+
+
+def test_static_job(tmp_path, monkeypatch):
+    monkeypatch.chdir(tmp_path)
+
+    copy_decompress_files([DATA_DIR / "H.usp.gz"], tmp_path)
+
+    keywords = {
+        "output_detail": "verbose",
+        "do_properties": True,
+        "cutoff_energy": "300 eV",
+        "task": "SinglePoint",
+        "pseudo_path": str(tmp_path),
+    }
+    pseudopotentials = {"H": "H.usp"}
```

Comment:
Most of these parameters are already in `static_job`, so please remove the redundant ones that are not being overriden.